class.jetpack-autoupdate.php

```diff
@@ -104,19 +152,26 @@ function expect( $item, $type='plugin' ) {
 	 *      'site_is_version_controlled'    => (bool) is the site under version control
 	 * )
 	 */
-	function save_update_data() {
+	function save_update_data( array $new_updates ) {
 		global $wp_version;
 
-		$update_data = wp_get_update_data();
+		$defaults = array(
```

Comment:
Instead of recalculating everything we could merge whatever is in `Jetpack_Options::update_option( 'updates' `) with the new value. What do you think @enejb?